jwcrypto/jwa.py

```diff
@@ -887,18 +887,18 @@ def __init__(self, hashfn):
         self.blocksize = algorithms.AES.block_size
         self.wrap_key_size = self.keysize * 2
 
-    def _mac(self, k, a, iv, e):
-        al = _encode_int(_bitsize(a), 64)
+    def _mac(self, k, ai, iv, e):
+        al = _encode_int(_bitsize(ai), 64)
         h = hmac.HMAC(k, self.hashfn, backend=self.backend)
-        h.update(a)
+        h.update(ai)
         h.update(iv)
         h.update(e)
         h.update(al)
         m = h.finalize()
         return m[:_inbytes(self.keysize)]
 
     # RFC 7518 - 5.2.2
-    def encrypt(self, k, a, m):
+    def encrypt(self, k, ai, m):
```

Comment:
this "ai" and others need to be renamed to "aad" too, see lint failures indicating it.

Reply:
no hurry

Reply:
Done, tested locally with pytest and pylint. it looks good to me